### PR TITLE
Update Go version to 1.21 in workflows and Dockerfiles

### PR DESCRIPTION
**Pull request by skrashevich**

### CODE REVIEW

File: hardware.Dockerfile [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 # 0. Prepare images
 # only debian 12 (bookworm) has latest ffmpeg
-ARG DEBIAN_VERSION="bookworm-slim"
-ARG GO_VERSION="1.20-buster"
+ARG DEBIAN_VERSION="bullseye-slim"
```
Comment (jmtatsch) [inline hardware.Dockerfile:5]: why go back to bullseye if only bookworm has the latest ffmpeg?

Comment (skrashevich) [reply hardware.Dockerfile]: make sense

Comment (jmtatsch) [reply hardware.Dockerfile]: thanks for advancing some of my favourite projects

**Comment by AlexxIT** [comment]
Thanks!

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/go2rtc/releases/tag/v1.7.0

**Pull request merged**